### PR TITLE
erlang 19.3

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by ilovezfs — COMMENTED** [review]
note: we need to wait for the new resources to be posted and the release to be announced before merging this

**Review by ilovezfs — COMMENTED** [review]
release has been announced now

File: Formula/erlang.rb [hunk 1]
```diff
@@ -34,13 +31,15 @@ class Erlang < Formula
...
 
   resource "html" do
-    url "https://www.erlang.org/download/otp_doc_html_19.2.tar.gz"
-    sha256 "c373c8c1a9fe7433825088684932f3ded76f53d5b8a4d3d2a364263f1f783043"
+    url "https://www.erlang.org/download/otp_doc_html_19.3.tar.gz"
+    mirror "https://fossies.org/linux/misc/otp_doc_html_19.3.tar.gz"
+    sha256 "dc3e3a82d1aba7f0deac1ddb81b7d6f8dee9a75e1d42b90c677a2b645f19a00c"
   end
 
   def install
```
Comment (fxcoudert) [inline Formula/erlang.rb]: Should the `_clock_gettime` hack be removed now? It seems it was merged.

Comment (ilovezfs) [inline Formula/erlang.rb]: let's find out

**Review by fxcoudert — APPROVED** [review]

**Comment by JCount** [comment]
@ilovezfs a 🍌 for you

**Pull request closed without merging**